resources/js/components/categoryComponent.vue

```diff
@@ -119,10 +119,10 @@ export default {
     },
     deleteCategory : function(id){
       let currentObj = this;
-      if(confirm("are your sure to delete the category?")){
+      if(confirm("are your sure to delete the product?")){
```

Comment:
that you want to

Reply:
Tant qu'à être la dedans, les messages sont en anglais?

Reply:
J'ai pas tout vérifier le code au niveau du front end, je vais regarder si il y aurait pas un moyen d'apporter i18n là dedans.